.prettierrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  printWidth: 120,
+  arrowParens: 'always',
+  tabWidth: 2,
+  semi: true,
+  bracketSameLine: true,
+  jsxSingleQuote: true,
+  singleQuote: true,
+  useTabs: false,
+  trailingComma: 'none'
```

Comment:
Could you please remove this line, and re-format the code? We use trailing commas by default